src/components/ActionControls/ActionControls.js

```diff
@@ -64,11 +64,13 @@ class ActionControls extends React.Component<Props, State> {
     onCreate = ({ text }: { text: string }) => {
         const { onCreate, type } = this.props;
         onCreate(type, text);
+        this.setState({ isInputOpen: false });
```

Comment:
Is there significance to setting state before/after the callback? It's set before in create and after in delete. Can it be standardized?

Reply:
Nope no significance. Moving it to before the callback so it's standardized